cli/src/pcluster/models/param.py

```diff
@@ -27,3 +27,21 @@ def __init__(self, value, default=None, valid: bool = True):
             self.value = value
             self.implied = False
         self.valid = valid
+
+    def __bool__(self):
```

Comment:
I'm not sure about these facilities because sometimes you need to call the param.value but for comparison you don't need it, I'd prefer an unambiguous behaviour.

Reply:
Done